src/korma/core.clj

```diff
@@ -548,10 +557,16 @@
    :rpk (raw (eng/prefix child (:pk child)))
    :join-table join-table})
 
-(defn- get-db-keys [parent child {:keys [fk]}]
-  (let [fk-key (or fk (default-fk-name parent))]
-    {:pk (raw (eng/prefix parent (:pk parent)))
-     :fk (raw (eng/prefix child fk-key))
+(defn- to-raw-key [ent k]
+  (if (seq? k)
+    (map #(raw (eng/prefix ent %)) k)
```

Comment:
here the to-raw-key function handles convertion a seq of keys (composite pk-key or composite fk-key)